llvm/lib/Target/AMDGPU/AMDGPUArgumentUsageInfo.h

```diff
@@ -161,14 +232,27 @@ struct AMDGPUFunctionArgInfo {
   ArgDescriptor WorkItemIDZ;
 
   // Map the index of preloaded kernel arguments to its descriptor.
-  SmallDenseMap<int, KernArgPreloadDescriptor> PreloadKernArgs{};
+  SmallDenseMap<int, KernArgPreload::KernArgPreloadDescriptor>
+      PreloadKernArgs{};
```

Comment:
```suggestion
      PreloadKernArgs;
```

Reply:
The constructor for `SmallDenseMap` is `explict` so I need to directly initialize it.